src/esm_runscripts/oasis.py

```diff
@@ -497,7 +497,7 @@ def add_restart_files(self, restart_file_label, fconfig):
                 # check if restart file with ini_restart_date in filename is in the restart
                 # folder of the parent experiment to be branched off from:
                 glob_search_file = (
-                    f"{restart_file_path}_????????-"
+                    f"{restart_file_path}*"
```

Comment:
What about the following? Then we are enforcing 2 dates format.
```suggestion
                    f"{restart_file_path}*_????????-"
```

Reply:
Yes that would work, but I'm assuming the last minus sign is a typo? 
```python
f"{restart_file_path}*_????????"
```
rather than
```python
f"{restart_file_path}*_????????-"
```

Reply:
Yes, the minus and the end must be a type :-)
And yes enforcing an 8 digit (YYYYMMDD) end date is probably better, but it assumes that people use YYYMMDD as date format.

Reply:
Hi,
the minus is not a typo, because the whole filename of the restart file is composed like this:
glob_search_file = (
```
f"{restart_file_path}_????????-"
f"{config['ini_restart_date'].year}"
f"{config['ini_restart_date'].month:02}"
f"{config['ini_restart_date'].day:02}"
 )
```
It seraches for a file with this pattern: `<restart_file_path>_????????-20000101`
For example:
`rstas.nc_20000101-20000131`

This pattern can be made more general for the restart file name as mentioned by @mandresm and @seb-wahl 
```
f"{restart_file_path}*_????????-"
f"{config['ini_restart_date'].year}"
f"{config['ini_restart_date'].month:02}"
f"{config['ini_restart_date'].day:02}"
 )
```